plugin/bundleVisualizer.ts

```diff
@@ -65,10 +65,23 @@ async function bundleVisualizer(this: ServerlessAnalyzeBundlePlugin): Promise<vo
 
   await pExec(`unzip .serverless/${functionZipName} -d ${TEMP_DIR_LOCATION}`);
 
-  const test = getAllFiles(`${TEMP_DIR_LOCATION}`);
-  const metafileName = test.filter(
-    fileName => fileName.includes(`/${functionName}/`) && fileName.endsWith('-meta.json'),
-  )[0];
+  const allFiles = getAllFiles(`${TEMP_DIR_LOCATION}`);
+  const metafileName = allFiles.filter(fileName => {
+    const handlerPath = (slsFunction as FunctionDefinitionHandler).handler.split('.')[0];
```

Comment:
will return the value configured in `serverless.yml` for the function handler, e.g. `src/handlers/foo.main`. We use split to get just up to the `.` as the part that follows is the exported function name